id/community/ruby-core/index.md

```diff
@@ -4,141 +4,168 @@ title: "Ruby Core"
+gunakan *branch* {{ site.svn.stable.branch }} ketika sedang *checkout*:
+
+{% highlight sh %}
+$ svn co https://svn.ruby-lang.org/repos/ruby/branches/{{ site.svn.stable.branch }}
+{% endhighlight %}
 
-Kalau Anda berminat mempatch Ruby 1.8, gunakan *branch* `ruby_1_8`
-ketika svn checkout:
+Demikian pula untuk Ruby {{ site.svn.previous.version }}:
 
 {% highlight sh %}
-$ svn co https://svn.ruby-lang.org/repos/ruby/branches/ruby_1_8
+$ svn co https://svn.ruby-lang.org/repos/ruby/branches/{{ site.svn.previous.branch }}
 {% endhighlight %}
 
-Perintah tadi untuk *checkout* direktori pengembangan Ruby versi 1.8.
-Developer yang bekerja di Ruby 1.8 diharapkan melakukan migrasi
-perubahan mereka ke trunk Ruby, sangat sering dua branch kelihatan
-sangat mirip, dengan pengecualian adanya perbaikan yang dibuat oleh Matz
-dan Nobu untuk bahasa Ruby sendiri.
+Perintah di atas akan melakukan *checkout* masing-masing ke direktori
+`{{ site.svn.stable.branch }}` or `{{ site.svn.previous.branch }}`.
+Pekerjaan pengembang pada *maintenance branch* diharapkan untuk migrasi
+perubahannya ke Ruby *trunk*, seringkali *branch-branch* kelihatan sangat
+mirip, dengan pengecualian ada perbaikan yang dibuat oleh Matz dan Nobu untuk
+bahasa Ruby sendiri.
 
-Kalau Anda berminat *browsing*, Anda bisa *browse* [repository Ruby via
+Kalau Anda berminat *browsing*, Anda bisa *browse* [repository Ruby melalui
 web][2].
 
 Untuk informasi lebih lanjut tentang Subversion, silakan lihat [the
 Subversion FAQ][3] dan [the Subversion book][4]. Sebagai alternatif,
 Anda bisa juga dapatkan buku [Pragmatic Version Control with Subversion][5]
 sebagai buku yang berguna untuk pengenalan svn.
 
+### Bagaimana Menggunakan Git pada Repositori Utama Ruby
+{: #git-ruby}
+
+Bagi mereka yang lebih memilih menggunakan [Git][6] dibanding Subversion dapat
+mengikuti instruksi di [mirror on GitHub][7], baik untuk [comitter][8]
+maupun [bukan commiter][9].
+
 ### Menyempurnakan Ruby, dengan Patch
 {: #patching-ruby}
 
-Rubyforge mempunyai [bug tracker][6] dengan submit patch dan bug report
-(laporan adanya bug) ke Matz dan rekan-rekan lainnya (Developer Ruby
-Core). Laporan-laporan ini juga di submit ke
-[milis Ruby-Core][mailing-lists] untuk didiskusikan,
-jadi Anda bisa mengkonfirmasi terlebih dahulu apakah permintaan Anda
-akan diperhatikan atau tidak. Anda juga bisa mengirim patch langsung ke
-milis. Anda diharapkan untuk berpartisipasi aktif dalam diskusi
-tersebut.
+Tim Ruby Core merawat [bug tracker][6] untuk memasukkan *patch* dan laporan
+*bug* ke Matz dan rekan-rekannya. Laporan-laporan ini juga dimasukkan ke
+[milis Ruby-Core][mailing-lists] untuk didiskusikan, jadi Anda bisa
+mengkonfirmasi terlebih dahulu apakah permintaan Anda akan diperhatikan atau
+tidak. Anda juga bisa mengirim *patch* langsung ke milis. Anda diharapkan
+untuk berpartisipasi aktif dalam diskusi tersebut.
+
+Mohon lihat [Patch Writer's Guide][writing patches] untuk beberapa tips,
```

Comment:
writing-patches

Reply:
Thanks for catching that. Just, fixed that line.